docs/cn/Debugging-Guide.md

```diff
@@ -19,6 +19,19 @@ Alluxio运行过程中可产生master、worker和client日志，这些日志存
 
 master和worker日志对于理解Alluxio master节点和worker节点的运行过程是非常有帮助的，当Alluxio运行出现问题时，可以查阅日志发现问题产生原因。如果不清楚错误日志信息，可在[邮件列表](https://groups.google.com/forum/#!forum/alluxio-users)查找，错误日志信息有可能之前已经讨论过。
 
+## Alluxio远程调试
+
+Alluxio一般不在开发机上运行,这使得Alluxio的调试变得困难,我们通常会用"增加日志-编译-部署运行-查看日志"的方法来定位问题,而这种定位问题的效率比较低而且需要修改代码从新部署,这在有些时候是不允许的。
+
+使用java远程调试技术可以简单、不修改源码的方式，进行源码级调试。你需要增加jvm 远程调试参数，启动调试服务。增加远程调试参数的方法有很多，比较方便的一种方法是，在需要调试的节点上，修改alluxio-env.sh，增加如下配置属性。
```

Comment:
Same here as my comment in https://github.com/Alluxio/alluxio/pull/4537, 
<pre>`alluxio-env.sh`</pre>